### ISSUE TITLE
Consider using an open source Git project platform

### ISSUE THREAD

**Issue by janvlug** [opensourcepledge/opensourcepledge.com#275]
Currently opensourcepledge is hosted on Github. Github is not free and open source software. Please consider moving to a free and open source hosting provider.

See for example:
https://docs.codeberg.org/getting-started/what-is-codeberg/#alternatives-to-codeberg

**Comment by vladh** [comment]
I agree and would prefer to use Codeberg. One difficulty is that we depend on some GitHub-specific features right now, such as the "Projects" section, mainly in a private repo.

**Comment by Delta456** [comment]
It is also easier for other users to collaborate on GitHub due to the ease. In future we can consider making this one a mirror of the Codeberg.

**Comment by vladh** [comment]
@Delta456 I don't agree. I understand that some people find PR-based workflows easier then, say, the email-based workflow of Sourcehut. But Codeberg's functionality on the contribution side is extremely similar to GitHub. In fact, you could make an argument that CodeBerg is easier to use because of its increased performance over GitHub: https://forgeperf.org/

**Comment by vladh** [comment]
Now is a good time to do this. Started work, and a task list, in #506.
